File: Training_Assistant/TraingAssistant/Models/Training.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TraingAssistantDAL.Models
{
    public class Training
    {
        public int Id { get; set; }
        public String Name { get; set; }
        public int Days { get; set; }
        public ICollection<Exercise>? Exercises { get; set; } = new List<Exercise>();
        public TrainingPlan? TrainingPlan { get; set; } 
        public int? TrainingPlanId { get; set; }
    }
}

File: Training_Assistant/TrainingAssistantFrontend/src/app/training/model/training.ts
import { Exercise } from "src/app/exercise/model/exercise";

export class Training{
    id:number;
    name:string;
    days:number;
    exercises:Exercise[];

    constructor(id:number,name:string,days:number,exercises:Exercise[]){
        this.id=id;
        this.name=name;
        this.days=days;
        this.exercises=exercises;
    }
}